pages/apim/3.x/installation-guide/docker/installation-guide-docker-compose.adoc

```diff
@@ -1,32 +1,26 @@
-= Docker Compose Installation
+= Custom Install with Docker Compose
 :page-sidebar: apim_3_x_sidebar
 :page-permalink: apim/3.x/apim_installation_guide_docker_compose.html
 :page-folder: apim/installation-guide/docker
 :page-layout: apim3x
 :page-description: Gravitee.io API Management - Installation Guide - Docker - Compose
 :page-keywords: Gravitee.io, API Management, apim, guide, manual, docker, compose, linux
-:page-toc: false
 :page-liquid:
 :table-caption!:
 
-// author: Tom Geudens
+This page describes how to install and run APIM in Docker containers on `localhost` using the `docker compose` command and a specified filesystem for persistence and plugins. 
 
-== Setup
-Before you can get into the actual installation, a couple of things need to be set up
+== Prerequisites
 
-. *Docker*
-+
-Is assumed to be installed.
+Docker must be installed and running. For more information about installing Docker, see the link:{{'https://www.docker.com/' | relative_url }}[Docker website].
 
-. *Docker Compose*
-+
-Is assumed to be installed.
+== Installing APIM
 
-. *Persistence*
-+
-Containers are great, but you do want persistence in case they need to be restarted. You are going to persist the MongoDB data, the Elasticsearch data and provide locations for additional plugins (both for the Gateway and the REST API). It is assumed in this page that these will live under a filesystem mounted at `/gravitee` on the host system. 
+=== Create the filesystem and download the Docker compose file
+
+1. Create the following filesystem.
 +

```

Comment:
I think we should provide the user with this command to create the filesystem so the user can just copy and paste. The current setup requires each line to be run separately and in order.
```
mkdir -p gravitee/{mongodb/data,elasticsearch/data,apim-gateway/plugins,apim-gateway/logs,apim-management-api/plugins,apim-management-api/logs,apim-management-ui/logs,apim-portal-ui/logs,installation}
```

Reply:
Nice suggestion! I have made this change throughout.